### DISCUSSION TITLE
Webhook and url?

### DISCUSSION THREAD

**Discussion by biswa4035** [MrMissx/Telegram_Forwarder#42] [Q&A]
What we have to enter in webhook and url while depolying in heroku?
I tried https://(botname).herokuapp.com/ as URL and WEBHOOK as ANYTHING, ENV as ANYTHING.
It gives error while the bot strts up.

**Comment by MrMissx** [comment]
You don't have to add that variable if your bot doesn't run in webhook.
and don't add https://(yourBotApp).herokuapp.com/ because we don't use any app preview in there. Adding it to the variables will only make your bot failed to start up.

**Reply by biswa4035** [reply]
thnx..got ur point. Bot is finally started. removed web url.
